cartridges/templates/wso2esb-4.8.1/configs.ini

```diff
@@ -0,0 +1,20 @@
+[SETTINGS]
+READ_FROM_ENVIRONMENT = false
```

Comment:
shall we make this to true by default since agent extension expose them as env varaiables